### PR TITLE
feat: Chart store UI enhancement 

### PR DESCRIPTION
**Pull request by milindk8**
# Description

Chart store 
1. Make list view as default
2. Grid/List option UI
3. Do not show GUI tab if GUI not available

Fixes # [ISSUE 1244](https://dev.azure.com/DevtronLabs/Devtron/_workitems/edit/1244)

image

image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/v2/values/chartValuesDiff/ChartValuesView.tsx [hunk 1]
```diff
@@ -920,22 +920,23 @@ function ChartValuesView({
     }
 
     const renderValuesTabs = () => {
+        const initialSelectedTab =
+            presetValueId || isCreateValueView
+                ? 'yaml'
```
Comment (vivek-devtron) [inline src/components/v2/values/chartValuesDiff/ChartValuesView.tsx:925]: Please use constant. And try to simplify this condition

Comment (milindk8) [reply src/components/v2/values/chartValuesDiff/ChartValuesView.tsx]: done

File: src/components/charts/ChartHeaderFilters.tsx [hunk 2]
```diff
@@ -126,31 +126,31 @@ function ChartHeaderFilter({
...
                 <div className="fs-12 h-36 pt-8 pb-8 cn-6 fw-6 ml-8">FILTERS</div>
                 <Checkbox
-                    rootClassName="cursor fs-13 bcn-0 ml-7 mr-10 mb-0 date-align-left--deprecate"
+                    rootClassName="fs-13 dc__hover-n50 pt-8 pb-8 pl-8 ml-8"
                     isChecked={includeDeprecated === 1}
                     value={'CHECKED'}
                     onChange={toggleDeprecated}
                 >
-                    <div className="ml-5"> Show deprecated charts</div>
+                    <div className=""> Show deprecated charts</div>
```
Comment (shivani170) [inline src/components/charts/ChartHeaderFilters.tsx:153]: Can remove this classname in case of empty

Comment (milindk8) [reply src/components/charts/ChartHeaderFilters.tsx]: done

File: src/components/v2/values/chartValuesDiff/ChartValuesView.tsx [hunk 3]
```diff
@@ -920,28 +920,27 @@ function ChartValuesView({
...
-                        ? 'gui'
-                        : 'yaml'
-                }
+                initialTab={initialSelectedTab.toLowerCase()}
                 disabled={false}
                 onChange={handleTabSwitch}
             >
-                <RadioGroup.Radio value="gui">GUI (Beta)</RadioGroup.Radio>
-                <RadioGroup.Radio value="yaml">
+                {initialSelectedTab === ConfigurationType.GUI && <RadioGroup.Radio value={ConfigurationType.GUI.toLowerCase()}>GUI (Beta)</RadioGroup.Radio>}
```
Comment (shivani170) [inline src/components/v2/values/chartValuesDiff/ChartValuesView.tsx:937]: Instead of GUI here also we can use `ConfigurationType.GUI.GUI`

**Review by vivek-devtron — APPROVED** [review]

**Review by shivani170 — APPROVED** [review]

**Pull request merged**